cmd/skaffold/app/cmd/config/config_test.go

```diff
@@ -217,23 +226,21 @@ func TestSetAndUnsetConfig(t *testing.T) {
 			// set specified value
 			err := setConfigValue(test.key, test.value)
 			actualConfig, cfgErr := readConfig()
-			if cfgErr != nil {
-				t.Error(cfgErr)
-			}
-			t.CheckErrorAndDeepEqual(test.shouldErrSet, err, test.expectedSetCfg, actualConfig)
+			t.CheckNoError(cfgErr)
+			t.CheckErrorAndDeepEqual(test.shouldErr, err, test.expectedSetCfg, actualConfig)
 
-			if test.shouldErrSet {
+			if test.shouldErr {
 				// if we expect an error when setting, don't try and unset
 				return
 			}
 
 			// unset the value
 			err = unsetConfigValue(test.key)
 			newConfig, cfgErr := readConfig()
```

Comment:
not an issue with this PR. But we shd de-couple testing set and unset behavior in the same test into 2 separate tests.